src/styles/components/blog/_BlogPost.scss

```diff
@@ -25,15 +25,11 @@
         }
 
         figcaption {
-            margin-top: 2rem;
+            margin: 1rem auto 3rem auto;
             text-align: center;
             font-style: italic;
         }
```

Comment:
I think we should use some util classes on the `figcaption` element and rm these global styles. Do you want to follow up with that in #5496 as cleanup and add the `Video` component to the other remaining places where `video` is used?

Reply:
Yes! I added a note in the issue to replace these with utilities. That extra consideration will help identify additional blog-related classes to break out into utilities, so that we embed some progress in prep for Tailwind ✨